tests/basalprofileplugin.test.js

```diff
@@ -0,0 +1,82 @@
+var should = require('should');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['should' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/771301346/issues/source?bid=2452379&fileBranchId=2479033#l1)